CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Add metric instrumentation for celery
+  ([#1679](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1679/files))
```

Comment:
Please remove the /files from the url
([#1679](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1679))